apps/jobs/models.py

```diff
@@ -70,9 +76,9 @@ class Submission(TimeStampedModel):
     started_at = models.DateTimeField(null=True, blank=True)
     completed_at = models.DateTimeField(null=True, blank=True)
     when_made_public = models.DateTimeField(null=True, blank=True)
-    input_file = models.FileField(upload_to=RandomFileName("submission_files/submission"))
-    stdout_file = models.FileField(upload_to=RandomFileName("submission_files/submission"), null=True, blank=True)
-    stderr_file = models.FileField(upload_to=RandomFileName("submission_files/submission"), null=True, blank=True)
+    input_file = models.FileField(upload_to=get_upload_path)
```

Comment:
Can we please remove `get_upload_path` and add some hook in `RandomFileName` class.